libaxolotl/test/temp_helpers.js

```diff
@@ -0,0 +1,85 @@
+var StaticByteBufferProto = new dcodeIO.ByteBuffer().__proto__;
```

Comment:
Hopefully...it means that it needs to shift to being a minimal helpers file in libaxolotl proper, and maybe shed some stuff, but that's probably longer term :(